client/src/core/Types.res

```diff
@@ -2,6 +2,13 @@ open Tc
 module PT = ProgramTypes
 module RT = RuntimeTypes
 
+// hmmm: are these 'client-only types'?
```

Comment:
I think we'd ideally like all these types to have there on file. It's a long process to get there though.